Makefile

```diff
@@ -10,13 +10,13 @@ all: format compile
 
 format:
 	@echo -e ":: $(GREEN)Checking TypeSpec format...$(NC)"
-	@tsp format "**/*.tsp" \
+	@npx tsp format "**/*.tsp" \
```

Comment:
Using `npx` here seems to pull `tsp@0.0.1` from npm, which is not the same as the `tsp` provided by `@typespec/compiler`. 

If we want to use `npx`, we need to explicitly specify the package like `npx -p @typespec/compiler tsp` to make sure the correct binary is used.